php-checks/src/main/java/org/sonar/php/checks/security/StandardInputUsageCheck.java

```diff
@@ -72,7 +72,11 @@ private void checkUsage(Tree tree) {
   }
 
   private static boolean isArgumentOfSafeFunctionCall(Tree tree) {
-    Tree parent = tree.getParent();
+    if (!tree.getParent().is(Tree.Kind.FUNCTION_CALL_ARGUMENT)) {
+      return false;
+    }
+
+    Tree parent = tree.getParent().getParent();
     if (parent.is(Tree.Kind.FUNCTION_CALL)) {
```

Comment:
Is this `if` condition still relevant?

Reply:
It wasn't but it is now after [this](https://github.com/SonarSource/sonar-php/pull/679/commits/3b3556ceca513c8fa36e64f0aa7e2bd2aab06208) as the parent can now also be an `AnonymousClass`.

Reply:
@pynicolas I've added a unit test to demonstrate the cause where it could go wrong when not verifying in this [commit](https://github.com/SonarSource/sonar-php/pull/679/commits/77b992795c911395999d63aee2134f87fad92743).